bndtools.core/src/bndtools/central/Central.java

```diff
@@ -726,4 +725,13 @@ public static void removeRepositoriesViewer(TreeViewer viewer) {
     public static void setRepositories(TreeViewer viewer, RefreshModel model) {
         repositoriesViewRefresher.setRepositories(viewer, model);
     }
+
+    public static ProjectPaths getProjectsPaths() {
+        try {
+            Workspace workspace = getWorkspace();
```

Comment:
Timing here may be bad. `getWorkspace` will force the workspace creation which we now try to avoid forcing. The previous code called a static `Workspace.getDefaults` method while this code will force the workspace creation. I think this is not good.

Reply:
This is called when the UI is active, i.e. when a new project is created. So this is long after activation. So I do not see the problem.

The whole idea is that we follow the Workspace.

Reply:
OK.